snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedDescription.java

```diff
@@ -227,6 +229,20 @@ public String getCaseSignificanceId() {
 		return acceptabilities;
 	}
 	
+	@JsonIgnore
+	public boolean isPreferredInLocales(final List<ExtendedLocale> locales) {
+		for (ExtendedLocale locale: locales) {
+			for (AcceptabilityMembership membership: acceptabilities) {
```

Comment:
`acceptabilites` field can be null if the client did not request it explicitly, however the `acceptabilityMap` is not null by default, I suggest falling back to the `acceptabilityMap` in case the `acceptabilities` field is `null`.